ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -157,9 +157,9 @@ objects:
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
         function fips_check() {
-          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
+          oc --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
           while IFS= read -r i; do
-            oc -n default --insecure-skip-tls-verify --request-timeout=60s debug node/$i -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
+            oc -n default --request-timeout=60s debug node/$i -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
```

Comment:
if i remember correctly,  i found that if i don't include the `-n something` w/ that debug node cmd, i see an error, something like `namespace ci-op-123 not found`